extensions/amp-story-360/validator-amp-story-360.protoascii

```diff
@@ -70,6 +70,18 @@ tags: {  # <amp-story-360>
     name: "scene-roll"
     value_regex: "-?\\d+\\.?\\d*"
   }
+  attrs: {
+    name: "scene-heading"
+    value_regex: "-?\\d+\\.?\\d*"
+  }
+  attrs: {
+    name: "scene-pitch"
+    value_regex: "-?\\d+\\.?\\d*"
+  }
+  attrs: {
+    name: "scene-roll"
+    value_regex: "-?\\d+\\.?\\d*"
+  }
```

Comment:
This diff is a duplicate of the above lines. It's a bug in `patch` during rollup. Revert these lines.

Reply:
Thanks, fixed.